patient-care/webapp/client/shared/editSampleGroup.js

```diff
@@ -51,6 +57,46 @@ Template.editSampleGroup.onRendered(function () {
 
   instance.$(".sample-group-version").popup({
     position : "bottom right",
+    content: "Sample group version",
+  });
+
+  instance.$(".data-set-search").search({
+    apiSettings: {
+      url: `${location.origin}/search/data-sets` +
+          `?token=${Accounts._storedLoginToken()}&q={query}`,
+      onResponse(response) {
+        // modify the response to remove existing data sets
```

Comment:
When a data set is added to the sample group & then removed again, the response doesn't refresh, so you can't see the data set in the "add a data set list" until you change what string you are searching for.